src/components/smart/Card.js

```diff
@@ -34,15 +34,18 @@ export class Card {
    * @param {Task} task
    * @param {string} title
    * @param {string} description
+   *
    * @returns {boolean}
    */
   editTask(task, title, description) {
     const i = this.taskList.indexOf(elem => elem === task);
     if (i !== -1) {
```

Comment:
empty row before if

Reply:
to clear code styles with if cases
// RIGHT
 public olololo(): void {
    if (true !== false) {
      if (false === false) {
        const kek: boolean = this.startToDoSmthg();

        if (kek) {
        .....
        }
      }
    }
  }


// WRONG check comments why wrong
 public olololo(): void {
 
if (true !== false) {      // it is the first statement in this lexical env, so there is no empty row
     
if (false === false) {     //same here
        const kek: boolean = this.startToDoSmthg();
        if (kek) {      // within one lexical env empty row if 'if' statement is going after smthg
        .....
        }
      }
    }
  }